database/iniciar_banco.sql

```diff
@@ -0,0 +1,12 @@
+CREATE SCHEMA roadmap;
+
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE roadmap.usuario(
+	nome varchar(100) NOT NULL,
+	email varchar(200) UNIQUE NOT NULL,
+	matricula varchar(11) UNIQUE NOT NULL,
+	data_nascimento date NOT NULL,
+	periodo smallint,
+	id uuid PRIMARY KEY DEFAULT uuid_generate_v4 ()
+);
```

Comment:
Comentário mais geral pra manter o padrão dos textos e arquivos em inglês, pra ficar padronizado.

Reply:
achei q o banco seria em portugues msm, acho q se for montar ele em ingles tem q mudar no diagrama tbm

Reply:
Acho melhor padronizar... Deixar tudo em inglês msm, fica mais fácil de associar as coisas